services/blockchainService.ts

```diff
@@ -20,12 +18,21 @@ export interface GetAddressParameters {
   reversedHashOrder?: boolean
 }
 
+export interface BlockchainInfoData {
```

Comment:
I think we could remove the `Data` from the name here and on the `BlockInfoData`, I think it is redundant with the `Info` part and it also goes together better with the method names: `getBlockInfo`, `getBlockchainInfo`

Reply:
true, addressed

Reply:
`BlockInfoData` still has the `Data` ending

Reply:
fixed